client/src/components/Container.jsx

```diff
@@ -51,15 +67,15 @@ class Container extends React.Component {
       });
   }
 
-  getAverageStars(storeId) {
-    console.log(this.state.reviews);
-    var average;
+  getAverageStars(reviews) {
     var total = 0;
```

Comment:
Recommend you get into the habit of using `let` or `const` only :)

Reply:
yeah this is something that I never got a full understanding of when and why to use const/let over var. can you help explain? or point me to a resource you might have used